samples/simple/conanfile.py

```diff
@@ -0,0 +1,34 @@
+# (C) Copyright 2025, SECO Mind Srl
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps
+from conan import ConanFile
+
+class Pkg(ConanFile):
+    # Binary configuration
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("astarte-device-sdk/0.7.0")
```

Comment:
Extract the version and set to a specific variable.
```suggestion
astarte_sdk_version = "0.7.0"      
        ...
        self.requires("astarte-device-sdk/astarte_sdk_version")
```